h/services/jwt.py

```diff
@@ -49,6 +50,8 @@ class JWTIssuers(StrEnum):
     """
 
     OIDC_CONNECT_OR_LOGIN_ORCID = "hak"
+    OIDC_REDIRECT_ORCID = "tir"
+    SIGNUP_VALIDATION_FAILURE_ORCID = "stol"
```

Comment:
These strings are so short that if you naively grep for "hak" or "tir" you'll find a bunch of other results as well.